ts/IterableInboxMessageDisplay.tsx

```diff
@@ -86,20 +96,24 @@ const IterableInboxMessageDisplay = ({
 
    const handleHTMLMessage = (event: any) => {
       let URL = event.nativeEvent.data
+
+      Iterable.trackInAppClick(rowViewModel.inAppMessage, IterableInAppLocation.inbox, URL)
+
       if(URL === 'iterable://delete') {
          deleteRow(rowViewModel.inAppMessage.messageId)
          returnToInbox()
       } else if(URL === 'iterable://dismiss') {
          returnToInbox()
+         Iterable.trackInAppClose(rowViewModel.inAppMessage, IterableInAppLocation.inbox, IterableInAppCloseSource.link)
       } else {
          if(Iterable.savedConfig.urlHandler) {
             let action = new IterableAction("openUrl", URL, "")
             let source = IterableActionSource.inApp
             let context = new IterableActionContext(action, source)
 
             Iterable.savedConfig.urlHandler(event.nativeEvent.data, context)
```

Comment:
Use `URL` instead of `event.nativeEvent.data`?

Reply:
Yes. I will update.